src/utils/createDynamicAppAtScriptTag.ts

```diff
@@ -71,7 +72,9 @@ export const createDynamicAppAtScriptTag = async (
   // in production we use a region specific selector
   const mountTargetID = isDevelopment
     ? elementID
-    : `${elementID}-${lang}${theme ? `-${theme}` : ``}`
+    : `${elementID}-${lang}${theme ? `-${theme}` : ``}${
+        darkMode ? `-dark` : ``
+      }`
 
```

Comment:
this is probably overkill, but I think its worth potential regression testing saves